examples/stable/test/property/candid_rpc/canister_methods/pre_upgrade/test/test.ts

```diff
@@ -76,10 +76,11 @@ const CanisterConfigArb = fc
         fc
             .array(HeterogeneousQueryMethodArb, small)
             .chain((queryMethods) =>
-                fc.constant([
-                    generateGetPreUpgradeExecutedCanisterMethod(),
-                    ...queryMethods
-                ])
+                fc.constant(
```

Comment:
Use the simplest generic to remove the readonly output of fc.constant

Reply:
Done